deployment_scripts/linode-marketplace-influxdb/influxdb-deploy.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+set -e
+trap "cleanup $? $LINENO" EXIT
+
+## InfluxDB Settings
+#<UDF name="soa_email_address" label="Email address (for the Let's Encrypt SSL certificate)" example="user@domain.tld">
+
+## Linode/SSH Security Settings
+#<UDF name="user_name" label="The limited sudo user to be created for the Linode: *No Capital Letters or Special Characters*">
+#<UDF name="disable_root" label="Disable root access over SSH?" oneOf="Yes,No" default="No">
+#<UDF name="pubkey" label="The SSH Public Key that will be used to access the Linode (Recommended)" default="">
```

Comment:
remove pubkey udf

Reply:
removed in [0b8e27b](https://github.com/akamai-compute-marketplace/marketplace-apps/pull/120/commits/0b8e27bb9b6641c2719566c7c337299997d95b54)